code/src/main/java/com/onlinecontacttracing/storage/data-storage/PositiveUserPlaces.java

```diff
@@ -37,4 +37,12 @@ public long getTimeCreatedSeconds() {
     return timeCreatedSeconds;
   }
 
+  public void add(String placeId, String name, long intervalStart, long intervalEnd) {
+    listOfPlaces.add(new Place(placeId, name, intervalStart, intervalEnd));
+  }
+
+  @Override
+  public String toString() {
+    return listOfPlaces.toString();
```

Comment:
This should probabl also include the user id.

Reply:
okay